download_certificates.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+strt=$(date +%s)
+
+home="https://infrastructure.fedoraproject.org/infra/rabbitmq-certs/"
+stag="staging/"
+prod="production/"
+
+mkdir /etc/firmitas/
+
+cat "certificates.txt" | while read line
```

Comment:
Hmm rather than use this certificates.txt file might be good to use the source of truth at https://infrastructure.fedoraproject.org/infra/rabbitmq-certs/production/expiration.txt instead?

Reply:
Source of truth for the list of certificates to go through or for the expiration dates? If it is for the first then I am afraid that there would be some processing required to filter out the dates part but if it is for the second then it removes the necessity of this project itself.

Reply:
https://gist.github.com/davidkirwan/5d9ef8de9bfe41f8e7eaeb3802e093cd

See this gist, uses the expiration.txt to dynamically create list of certs which it will then download into a ./certificates/production and ./certificates/staging directories.

this way we dont need to redploy firmitas when a new cert is created, it will automatically get picked up 24 hours later via the cronjob, deployed to https://infrastructure.fedoraproject.org/infra/rabbitmq-certs/ and then this download_certificates.sh script will pick it up.